back/docs/docs.go

```diff
@@ -1,6 +1,6 @@
 // GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
 // This file was generated by swaggo/swag at
-// 2020-03-09 23:02:41.469419863 +0100 CET m=+0.124852190
+// 2020-03-12 22:02:56.411483541 +0100 CET m=+0.081028564
```

Comment:
Sollen wir die Swagger generierten Files mal aus dem VCS rausnehmen? Generierung ist ja schon in der Pipeline.

Reply:
ja können wir als cleanup task in Jira mit aufnehmen